ideas/170-wallet-improvements.md

```diff
@@ -0,0 +1,74 @@
+## Preamble
+
+    Idea: 170
+    Title: Wallet Improvements
+    Status: Draft
+    Created: 2018-04-13
+    
+## Summary
+
+Status Wallet does not fully support ERC20 tokens (requesting and transaction history not available)
+and generally the transactions UX is different between Chat and Wallet. The intention of this swarm
+is to wrap up those inconsistencies.
+
+## Swarm Participants
+- Lead Contributor: [@goranjovic](https://github.com/goranjovic)
+- Contributor: 
+- Testing & Evaluation: [@asemiankevich] (https://github.com/asemiankevich)
+- PM: [@chadyj](https://github.com/chadyj)
+- UX: 
+
+## Product Overview & Description
+
+Add native support for all ERC20 token operations and rudimentary support for ERC721 tokens.
```

Comment:
Also UXR insight and UX improvements to the wallet experience?

Perhaps a survey similar to messaging as a benchmark that can be used to compare final results?
And specific wallet testing in the upcoming Berlin UXR sessions?